chainerx_build_helper.py

```diff
@@ -40,6 +54,15 @@ def run(self):
         if cmake_version < '3.1.0':
             raise RuntimeError('CMake >= 3.1.0 is required to build ChainerX')
 
+        self.use_ninja = False
+        prefer_ninja = get_env_boolean("CHAINERX_PREFER_NINJA", default="False")
```

Comment:
An alternative could be to introduce a `CHAINERX_CMAKE_GENERATOR` taking a generator name similar to [`CMAKE_GENERATOR`](https://cmake.org/cmake/help/latest/envvar/CMAKE_GENERATOR.html#envvar:CMAKE_GENERATOR) with similar behavior i.e. to fall back to system default if not specified/found. Maybe raising a warning would be friendly if it's specified but not found. We'd also reserve ourselves for future other generators besides Ninja.